code/game/objects/structures/props.dm

```diff
@@ -892,14 +892,44 @@
 	health = 30
 	var/inscription
 	var/obj/item/helmet
-
+	var/tagged = FALSE
+	var/engraved = FALSE
+	var/dogtag_name
+	var/dogtag_blood
+	var/dogtag_assign
 /obj/structure/prop/wooden_cross/Destroy()
 	if(helmet)
 		helmet.forceMove(loc)
 		helmet = null
+	if(tagged)
+		var/obj/item/dogtag/newInfoTag = new(loc)
+		newInfoTag.fallen_names = list(dogtag_name)
+		newInfoTag.fallen_assgns = list(dogtag_assign)
+		newInfoTag.fallen_blood_types = list(dogtag_blood)
+		fallen_list_cross.Remove(dogtag_name)
 	return ..()
 
 /obj/structure/prop/wooden_cross/attackby(obj/item/W, mob/living/user)
+	if(istype(W, /obj/item/dogtag))
+		var/obj/item/dogtag/D = W
+		if(!tagged)
+			tagged = TRUE
+			user.visible_message(SPAN_NOTICE("[user] drapes the [W] around the cross."))
+			dogtag_name = D.fallen_names[1]
+			dogtag_assign = D.fallen_assgns[1]
+			dogtag_blood = D.fallen_blood_types[1]
+			fallen_list_cross.Add(dogtag_name)
+			update_icon()
+			D.fallen_blood_types.Remove(dogtag_blood)
+			D.fallen_assgns.Remove(dogtag_assign)
+			D.fallen_names.Remove(dogtag_name)
+			if(D.fallen_names.len == 0)
+				qdel(D)
+			else
+				return
+		else
+			to_chat(user, SPAN_WARNING("There's already a dog tag on the [src]!"))
```

Comment:
```suggestion
			to_chat(user, SPAN_WARNING("There's already a dog tag on the [src]!"))
			balloon_alert(user, "already a tag here!")
```
you don't need to do this but it's nice visual feedback we should be adding to more places